cfme/tests/webui/test_general_ui.py

```diff
@@ -523,6 +527,6 @@ def test_custom_navigation_menu():
             1. Custom menu must be visible.
 
     Bugzilla:
-        * 1678151
+        1660460
```

Comment:
Looks like this number changes? is this intentional?

Reply:
Good catch, the original ID is the one we want!

Reply:
@john-dupuy called `Eagles eyes` :) I missed this...